news/2024/2024-04-17-osutaiko-world-cup-2024-semifinals-recap.md

```diff
@@ -0,0 +1,93 @@
+    </audio>
+</div>
+
+## Semifinals results
+
+Drum roll please... Yeah yeah, funny joke I know. Here are the results.
+
+| Team A |  |  | Team B |
+| --: | :-: | :-: | :-- |
+| **China** ::{ flag=CN }:: | **6** | 0 | ::{ flag=FI }:: Finland |
+| **Japan** ::{ flag=JP }:: | **6** | 0 | ::{ flag=BR }:: Brazil |
+| **Taiwan** ::{ flag=TW }:: | **6** | 0 | ::{ flag=CA }:: Canada |
+| **Chile** ::{ flag=CL }:: | **6** | 2 | ::{ flag=FR }:: France |
+| **United States** ::{ flag=US }:: | **6** | 1 | ::{ flag=MY }:: Malaysia |
+| **United Kingdom** ::{ flag=GB }:: | **6** | 5 | ::{ flag=KR }:: South Korea |
+| China ::{ flag=CN }:: | 3 | **6** | ::{ flag=TW }:: **Taiwan** |
+| **United States** ::{ flag=US }:: | **6** | 2 | ::{ flag=CL }:: Chile |
+
+## Match highlights
+
+### ::{ flag=GB }:: United Kingdom vs. ::{ flag=KR }:: South Korea
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2119605829&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+There's South Korea. Former podium-placers. A team with several top 50 players and tournament veterans, well-rounded and fit to cover everything the mappools have to offer. Seeding 3rd with balanced placements across the board. Then there's the United Kingdom, the former podium-placers. A team with top 50 players and tournament veterans, well-rounded and fit to…
+
+Hey… Wait a minute… Aren't they basically the same? Looking at the stats, the only thing separating these guys' seedings is a rank sum difference of *two*. And you're telling me they're up against each other now? In the winners' bracket semifinals?! In a battle to secure the aforementioned podium placement?! (Localized entirely within your kitchen?!)
+Well you better strap in, because this was exactly the type of roller coaster match you're imagining.
+
+The United Kingdom won the roll and came swinging right out of the gate with [Masayoshi Iimori - NEWALONE (Extended Mix)](https://osu.ppy.sh/beatmapsets/2163092#taiko/4561817), grabbing a quick point thanks to their spectacular accuracy. South Korea countered with [MYUKKE. - Mach Roger](https://osu.ppy.sh/beatmapsets/2163105#taiko/4561831), thinking they could make use of their Hidden trump card, ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027/taiko), but nuh-uh. The UK grabbed that one too, and continued to do so with two more picks, making it 4–0. A gut-wrenching sight for the Koreans.
+
+The United Kingdom had proven themselves to be *just a cut above*, but when they went fishing in the Free Mod pool with [Sparxe - Switcheroo](https://osu.ppy.sh/beatmapsets/2163116#taiko/4561848), South Korea found the opening they'd been looking for, grabbing a breakpoint and bouncing back with one after another. Some time passed and all of a sudden we were here again, the first tiebreaker since the Round of 32.
```

Comment:
```suggestion
The British had proven themselves to be *just a cut above*, but when they went fishing in the Free Mod pool with [Sparxe - Switcheroo](https://osu.ppy.sh/beatmapsets/2163116#taiko/4561848), the South Koreans found the opening they'd been looking for, grabbing a breakpoint and bouncing back with one after another. Some time passed and all of a sudden we were here again — the first tiebreaker since the Round of 32.
```